docs/deployment/contract-deployment-options.md

```diff
@@ -0,0 +1,114 @@
+---
+sidebar_position: 1.4
+title: Contract Deployment Options
+---
+
+When using `LSPFactory` you have the ability to specify custom deployment parameters to be applied to your contracts.
+
+The `deploy` function takes an object `contractDeploymentOptions` as its second parameter where the contract `version`, `byteCode` and `libAddress` can be specified:
+
+> `contractDeploymentOptions?` - `Object`:
```

Comment:
Maybe putting a JS code snippet of the `deploy(...)` function with the `contractDeploymentOptions` object as a parameter would explain it better, and make it more intuitive?

```js
LSPFactory.deploy({
    version: "1.0",
    byteCode: "0x6080604052..........",
    deployProxy: false,
    libAddress: "..."
});
```
 or 
 
 ```js
 const contractDeploymentOptions = {
    version: "1.0",
    byteCode: "0x6080604052..........",
    deployProxy: false,
    libAddress: "..."
};

LSPFactory.deploy(contractDeploymentOptions);
```